### PR TITLE
RHDEVDOCS-4445 - Added 1.6.1, 1.5.6 and 1.4.12 release notes

### PR DESCRIPTION
**Pull request by DebarghoGhosh**
Aligned team: Dev Tools
OCP version for cherry-picking: enterprise- 4.8 - 4.12

JIRA issue:
https://issues.redhat.com/browse/RHDEVDOCS-4445

Preview pages: https://50294--docspreview.netlify.app/openshift-enterprise/latest/cicd/gitops/gitops-release-notes.html

SME+QE review: @wtam2018 
Peer-review: @gabriel-rh

### CODE REVIEW

File: modules/gitops-release-notes-1-4-12.adoc
Comment (wtam2018) [inline modules/gitops-release-notes-1-4-12.adoc:23]:
```suggestion
* Before this update, Dex pods failed to start with CreateContainerConfigError when a SCC of 'anyuid' is assigned to the Dex ServiceAccount. This update fixes this issue by assigning a default user id to the Dex container.
  link:https://issues.redhat.com/browse/GITOPS-2235[GITOPS-2235] 
```

File: modules/gitops-release-notes-1-4-12.adoc
Comment (wtam2018) [inline modules/gitops-release-notes-1-4-12.adoc:20]: please include the details and instructions outlined in the  Release notes text of  https://issues.redhat.com/browse/GITOPS-2214

**Review by wtam2018 — COMMENTED** [review]
LGTM

File: modules/gitops-release-notes-1-4-12.adoc
Comment (gabriel-rh) [inline modules/gitops-release-notes-1-4-12.adoc:17]:
```suggestion
* Before this update, in a large set of applications, the application controllers were restarted multiple times due to the unresponsiveness of liveness probes. This update fixes the issue by removing the liveness probe in the application controller `StatefulSet`.  link:https://issues.redhat.com/browse/GITOPS-2153[GITOPS-2153]
```

File: modules/gitops-release-notes-1-4-12.adoc
Comment (gabriel-rh) [inline modules/gitops-release-notes-1-4-12.adoc:15]: Personally, I am happy with the passive voice here.

But if you want to avoid it, as per the style guide, you could say:

"The current release resolves the following issues:"

File: modules/gitops-release-notes-1-4-12.adoc
Comment (gabriel-rh) [inline modules/gitops-release-notes-1-4-12.adoc:19]:
```suggestion
* Before this update, the RHSSO certificate cannot be validated when it is set up with a certificate which is not signed by certificate authorities. This update fixes the issue and now you can provide a custom certificate which will be used in verifying the Keycloak's TLS certificate when communicating with it. You can add the `rootCA` to the Argo CD custom resource `.spec.keycloak.rootCA` field. The Operator reconciles this change and updates the `oidc.config` field in the `argocd-cm` `ConfigMap` with the PEM-encoded root certificate. 
```

File: modules/gitops-release-notes-1-4-12.adoc
Comment (gabriel-rh) [inline modules/gitops-release-notes-1-4-12.adoc:23]:
```suggestion
Restart the Argo CD server pod after updating the `.spec.keycloak.rootCA` field.
```

File: modules/gitops-release-notes-1-4-12.adoc
Comment (gabriel-rh) [inline modules/gitops-release-notes-1-4-12.adoc:52]:
```suggestion
* Before this update, a terminating namespace that was managed by Argo CD would block the creation of roles and other configuration of other managed namespaces. This update fixes this issue. link:https://issues.redhat.com/browse/GITOPS-2277[GITOPS-2277]
```

File: modules/gitops-release-notes-1-4-12.adoc
Comment (gabriel-rh) [inline modules/gitops-release-notes-1-4-12.adoc:54]:
```suggestion
* Before this update, the Dex pods failed to start with `CreateContainerConfigError` when an SCC of `anyuid` was assigned to the Dex `ServiceAccount` resource. This update fixes this issue by assigning a default user id to the Dex container. link:https://issues.redhat.com/browse/GITOPS-2235[GITOPS-2235] 
```

File: modules/gitops-release-notes-1-5-6.adoc
Comment (gabriel-rh) [inline modules/gitops-release-notes-1-5-6.adoc:54]: This looks like a copy/paste error

**Review by gabriel-rh — COMMENTED** [review]
Looks good - the suggestions apply to all three sets of RNs.

The copy/paste error needs to be fixed.

File: modules/gitops-release-notes-1-4-12.adoc
Comment (DebarghoGhosh) [reply modules/gitops-release-notes-1-4-12.adoc:15]: @gabriel-rh keeping the passive voice, since all the previous release notes have the same.

**Review by gabriel-rh — APPROVED** [review]
Looks good to me. Thanks!

**Comment by bburt-rh** [comment]
@DebarghoGhosh There's a formatting indentation issue with the YAML sample code and the links to the issue in all of the files. This needs to be fixed before it can be merged. I'm pasting a screen grab into Slack to show you the issue.

**Comment by DebarghoGhosh** [comment]
> @DebarghoGhosh There's a formatting indentation issue with the YAML sample code and the links to the issue in all of the files. This needs to be fixed before it can be merged. I'm pasting a screen grab into Slack to show you the issue.
@bburt-rh changes done

**Comment by bergerhoffer** [comment]
@DebarghoGhosh @bburt-rh It looks like this PR made it to 4.8 but not 4.9. Should this still be manual CPd to 4.9?

**Comment by bburt-rh** [comment]
Thanks, @bergerhoffer, for following up. 

Last time I pinged @DebarghoGhosh on Slack, he indicated that "for 4.9 we are skipping as of now, there are some bigger changes for 4.9 release notes. Will merge when we will make the changes"

Debhargo - Is this still the status?

**Pull request merged**
